breadwallet/src/ApplicationController.swift

```diff
@@ -70,10 +65,19 @@ class ApplicationController : Subscriber, Trackable {
         updateAssetBundles()
     }
 
+    private func setupWallet() {
```

Comment:
Moved wallet initialization to separate function called from `setup` to avoid the possibility of `didInitWallet` being called before `window.rootViewController` was set.